staging/src/k8s.io/client-go/transport/token_source_test.go

```diff
@@ -211,16 +259,19 @@ func TestCancellation(t *testing.T) {
 	}
 }
 
-type recordCancelRoundTripper struct {
+type testTransport struct {
 	canceled bool
 	base     http.RoundTripper
 }
 
-func (rt *recordCancelRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+func (rt *testTransport) RoundTrip(req *http.Request) (*http.Response, error) {
+	if req.Header["Authorization"][0] == "Bearer bad" {
+		return nil, apierrors.NewUnauthorized("")
```

Comment:
this should return a 401 response and nil error

Reply:
done